Fix a broken link and build warning

adiati98 opened the pull request:
## Description

This PR holds below changes:

- Fixed a broken link and addresses a build warning
- Applied vale off/on statements in `config.rst`.


<!-- PLEASE WRITE ABOVE THIS COMMENT. -->

<!--

Clearly describe what changes you have made in this PR, where our maintainers or reviewers should look at, and how to test the changes.

If the request is not complete but you want feedback or have quetions, you can select the "Draft Pull Request" option from the dropdown menu when creating the PR, then ask your questions or write your feedback in the comment.

-->

## Linked issue

Closes #267


<!-- PLEASE WRITE ABOVE THIS COMMENT. -->

<!--

If your PR is related to a current issue, please link to that issue number. Type the keyword "Closes" followed by a hashtag (#) symbol and the issue number that you can find right after the issue title. Don't add anything else, such as period, comma, etc, to this. For example:

❌ Closes: #123.

✅ Closes #123

Doing so will automatically close the issue when one of our maintainers merges your PR.

-->

## Screenshots or screen recordings

<!-- 

Provide screenshots or screen recordings before and after the changes, if it's a UI changes. You can simply drag and drop your files above this comment.

-->

adiati98 committed:
fix broken link to mautic community handbook

adiati98 committed:
fix build warning
* apply vale statements in config.rst

RCheesley commented on docs/plugins/config.rst:
```diff
@@ -199,9 +201,13 @@ Configure custom routes through writing a listener to the ``\Mautic\CoreBundle\C
 
 .. php:method:: getType()
 
+.. vale off
```
I don't think you can use Vale within the php class/method - this text seems to be missing:

<img width="3456" height="1916" alt="image" src="https://github.com/user-attachments/assets/36c0ccad-fdad-445f-bd89-e59c4e5d758c" />

Probs need to use it before the method declaration?

adiati98 replied:
I checked my changes, and it's the same with the live website. So they all actually broken.
Maybe let's just remove the vale here. Might give some errors, but it's because of the code

adiati98 replied:
This is weird. I removed the vale statements completely from here, but the result is still the same as your screenshot.

adiati98 replied:
I'm not familiar with php.
But can you confirm that the whole block is php code block?

```php
.. php:class:: Mautic\CoreBundle\Event\RouteEvent

  .. php:method:: getType()

      :returns: The :ref:`route firewall<plugins/config:Routing firewalls>` for the given route collection.
      :returntype: string

  .. php:method:: getCollection()

      :returns: Returns a RouteCollection object that can be used to manually define custom routes.
      :returntype: \\Symfony\\Component\\Routing\\RouteCollection

  .. php:method:: addRoutes(string $path)

      Load custom routes through a resource file such as yaml or XML.

      :param string $path: Path to the resource file. For example, ``@FMElfinderBundle/Resources/config/routing.yaml``.
      :returntype: void
```

adiati98 replied:
If so, I think I know where it's gone wrong.

adiati98 replied:
I've removed the vale off/on. I'm not sure how these RST syntax should be rendered on the site, though. I'm asking confirmation from Rahul on Slack.

adiati98 replied:
The "rendering problem" has been fixed in https://github.com/mautic/developer-documentation-new/pull/269. 👍🏼

RCheesley reviewed (changes requested):
Just one part where I think you need to shift the Vale off/on outside the PHP method/class declarations as the content isn't showing.

adiati98 committed:
remove vale off/on in the middle of code block

adiati98 committed:
Merge branch '5.x' into fix-links

adiati98 committed:
remove all vale statements and excess empty lines

adiati98 commented:
@RCheesley I've removed all vale statements. They don't seem to yell here. 
The weird code rendered as [you mentioned](https://github.com/mautic/developer-documentation-new/pull/268#discussion_r2343797463) has been fixed in #269.

This PR now only holds below changes:

- Fixed a broken link that caused build fails.
- Removed excess empty lines.

So I think this is good to go.

RCheesley reviewed (approved):
GTG!

RCheesley commented:
Does this also need fixing in other branches @adiati98 ?

adiati98 commented:
I think we only have one branch in developer docs?
